versions.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+module-sets:
+  adot-base:
+    version: v0.14.0-RC1
```

Comment:
Why `v0.14.0-RC1`?  `v0.14.0` has already shipped, right?

Reply:
Interesting. I thought I would need to setup in the separate PR. Also another typo for me, the v.0.14.0 was shipped to cx.